server/routes/createTodos.js

```diff
@@ -0,0 +1,9 @@
+const { connect } = require('hyper-connect')
+require('dotenv').config()
+
+const hyper = connect(process.env.HYPER)
```

Comment:
consider moving your hyper instance into a separate module ie. `hyper.js`, then requiring that module in routes.